vaadin-grid-pro-flow-demo/src/main/java/com/vaadin/flow/component/gridpro/vaadincom/GridProView.java

```diff
@@ -68,6 +71,35 @@ private void editorTypes() {
         addCard("Editor Types", grid);
     }
 
+    private void customRepresentation() {
+        // begin-source-example
+        // source-example-heading: Custom Representation
+        GridPro<Person> grid = new GridPro<>();
+        grid.setItems(createItems());
+
+        /*
+         * Grid Pro is an extension of the Grid and provides all
+         * the same functionality on top of basic one.
+         * It is possible to use Grid's API in Grid Pro.
+         */
+        grid.addColumn(Person::getName).setHeader("NAME");
+
+        /*
+         * Using ComponentRenderer to create a custom representation of the boolean value.
+         */
+        ComponentRenderer<Span, Person> booleanRenederer = new ComponentRenderer<>(person -> {
+            if (person.isSubscriber()) {
+                return new Span("Yes");
+            } else {
+                return new Span("No");
+            }
+        });
+        grid.addComponentEditColumn(Person::isSubscriber, booleanRenederer).checkbox((item, newValue) -> {}).setHeader("Subscriber (editable)");
```

Comment:
This should be broken to multiple lines.
At least in Flow team we had max line length 80 characters.
Should be configured in your IDE to wrap automatically.
Otherwise, make new lines start at `.checkbox...` and `.setHeader...`

Reply:
Updated.